### PR TITLE
Implements #2 - Adds t/h for VerticalRate and Altitude in reducer

### PR DESCRIPTION
**Pull request by tardoe**
As above.

### CODE REVIEW

File: cmd/pw_router/worker.go [hunk 1]
```diff
@@ -22,6 +22,8 @@ var (
 )
 
 const SigHeadingChange = 1.0 // at least 1.0 degrees change.
+const SigVerticalRateChange = 180.0 // at least 180 fpm change (3ft in 1min)
+const SigAltitudeChange = 10.0 // at least 5 ft in altitude change.
```
Comment (bluntelk) [inline cmd/pw_router/worker.go:26]: constant does not equal the comment

File: cmd/pw_router/worker.go [hunk 2]
```diff
@@ -50,7 +52,7 @@ func (w *worker) isSignificant(last export.PlaneLocation, candidate export.Plane
 		return true
 	}
 
-	if candidate.HasVerticalRate && last.HasVerticalRate && candidate.VerticalRate != last.VerticalRate {
+	if candidate.HasVerticalRate && last.HasVerticalRate && math.Abs(float64(candidate.VerticalRate-last.VerticalRate)) > SigVerticalRateChange {
```
Comment (bluntelk) [inline cmd/pw_router/worker.go:55]: Do we want to have this only above a certain altitude? 180ft could be a significant enough change below 2000ft for example?

**Review by bluntelk — CHANGES_REQUESTED** [review]
just a question and a comment

File: cmd/pw_router/worker.go (hunk 2 above)
Comment (tardoe) [reply cmd/pw_router/worker.go:55]: @bluntelk this is VerticalRate, as the altitude is changing closer to the ground, that will cause us to emit a significant frame. This is more to mute out "wobble" in fpm while a plane is in cruise. E.g. if a plane dips 2ft in the air during cruise, we get a frame with -120fpm. The 10ft of "SigAltitudeChange" will work with this too.

**Pull request merged**